.ci/build.ps1

```diff
@@ -0,0 +1,28 @@
+$ErrorActionPreference="Stop"
+
+$gradleInit = "C:\Users\$env:username\.gradle\init.d\"
+echo "Remove $gradleInit"
+Remove-Item -Recurse -Force $gradleInit -ErrorAction Ignore
```

Comment:
Ephemeral workers don't have anything there, but we still have a few workers that are not ephemeral, and we want to make sure we don't keep old scripts around. 
We have to use an init script as that's the only way it will be applicable to all invocations without a lot of boilerplate to pass that around.